BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs

```diff
@@ -24,4 +31,37 @@ public void Index_WhenCalledByAnonymousUser_ReturnsViewWithLoginLink()
         result!.ViewData.Model.Should().BeOfType<AdminViewModel>()
             .Which.CurrentUser.Should().BeNull();
     }
+
+    [Fact]
+    public void GetAdminList_ReturnsViewWith_AdminListModel()

```

Comment:
Don't really need the second underscore, reads oddly 

```suggestion
public void GetAdminList_ReturnsViewWithAdminListModel()
```